lib/linguist/languages.yml

```diff
@@ -5897,6 +5878,35 @@ Windows Registry Entries:
   codemirror_mode: properties
   codemirror_mime_type: text/x-properties
   language_id: 969674868
+Wolfram Language:
+  type: programming
+  extensions:
+  - ".cdf"
+  - ".m"
+  - ".ma"
+  - ".mt"
+  - ".nb"
+  - ".nbp"
+  - ".wl"
+  - ".wls"
+  - ".wlt"
+  filenames:
+  - init.m
+  - init.wl
+  aliases:
+  - Mathematica
+  - mma
+  - wl
+  interpreters:
+  - wolframscript
+  - wolfram
+  - math
+  - MathKernel
```

Comment:
I found many examples of `wolframscript` and `MathKernel` on github.com, but none for `wolfram` and `math`. Do users of these last two interpreters tend to use specific file extensions (I tried `.wls` and `.ma`)?

Reply:
As described [here](https://reference.wolfram.com/language/ref/program/wolfram.html), **wolfram** is a way to launch **WolframKernel** in a console, and in fact in some systems it is merely a symbolic link to the Kernel binary (e.g., MacOS, at least since Kernel version 12.3, but possibly earlier). The same is true for **MathKernel** in MacOS.

[This tech note](https://reference.wolfram.com/language/tutorial/LaunchingMathematicaOnLinux.html) clarifies that **math** is a way to initiate a Kernel session without a FrontEnd on Linux, so it is basically the same as WolframKernel or wolfram. [This other tech note](https://reference.wolfram.com/language/tutorial/LaunchingMathematicaOnWindows.html) says **math** played the same role on Windows in version 12. It may have been updated to **WolframKernel**/**wolfram** since then. On MacOS, as of version Kernel version 13, the **math** binary is not available (neither through Mathematica nor via the Wolfram Engine).